pkg/cli/flags.go

```diff
@@ -59,6 +59,11 @@ func (cl *CommandLineInterface) ByteQuantityMinMaxRangeFlags(name string, shorth
 	cl.ByteQuantityMinMaxRangeFlagOnFlagSet(cl.Command.Flags(), name, shorthand, defaultValue, description)
 }
 
+// Float64MinMaxRangeFlags creates and registers a min, max, and helper flag each accepting a float64
```

Comment:
super nit: Float64

Reply:
gotcha, I went back and removed arbitrary caps on the other mentions of types.